src/indexer/utils/evm/index.ts

```diff
@@ -194,19 +185,32 @@ export function parseProposalExecution(
   }
 }
 
-export async function parseFeeCollected(
-  decodedLog: LogDescription,
-  provider: Provider,
-  nativeTokenSymbol: string,
-  log: Log,
-): Promise<DecodedFeeCollectedLog> {
-  const ercToken = getERC20Contract(provider, decodedLog.args.tokenAddress as string)
-
-  return {
-    amount: BigNumber.from(decodedLog.args.fee as number).toString(),
-    tokenSymbol: (decodedLog.args.tokenAddress as string) == nativeTokenAddress ? nativeTokenSymbol : ((await ercToken?.symbol()) as string) || "",
-    tokenAddress: decodedLog.args.tokenAddress as string,
-    txHash: log.transactionHash,
+export async function getFee(provider: Provider, feeHandlerRouterAddress: string, fromDomain: Domain, decodedLog: LogDescription): Promise<FeeData> {
+  try {
+    const feeRouter = gerFeeRouterContract(provider, feeHandlerRouterAddress)
+    const fee = (await feeRouter.calculateFee(
+      decodedLog.args.user as string,
+      fromDomain.id,
+      decodedLog.args.destinationDomainID as string,
+      decodedLog.args.resourceID as string,
+      decodedLog.args.data as string,
+      "0x00",
```

Comment:
This will work when there is no fee data, but is not really a long term solution.

Reply:
Maybe it is fine for now, but create an issue to either use Fee_Collected again or parse fee data from deposit arguments.